### ISSUE TITLE
Directly including dependencies in your addon is a bad idea and can break other addons

### ISSUE THREAD

**Issue by 30350n** [CreativeDesigner3D/home_builder_4#6]
See [pcb2blender#57](https://github.com/30350n/pcb2blender/issues/57).

**Comment by CreativeDesigner3D** [comment]
Yeah. I have a couple python libraries that I use for reporting purposes. I'll need to research the best way to do this.

**Comment by 30350n** [comment]
If you are interested I've created [`blender_addon_utils`](https://github.com/30350n/blender_addon_utils) exactly for this purpose. It handles the installation of 3rd party libraries (it doesn't do it automatically, but the user has to manually confirm it).

**Comment by CreativeDesigner3D** [comment]
Thanks I will look into it. For now I have just removed the dependencies.
